distribution/lib/Standard/Base/0.0.0-dev/src/Runtime.enso

```diff
@@ -52,8 +53,20 @@ get_stack_trace =
 gc : Nothing
 gc = @Builtin_Method "Runtime.gc"
 
+## ADVANCED
+
+   Asserts that the given action succeeds, otherwise throws a panic.
+
+   Assertions are disable by default, meaning that call to this method is
+   a no-op. To enable assertions, set the environment variable
+   `ENSO_ENABLE_ASSERTIONS=true`
+assert : (Any -> Boolean) -> Text -> Nothing ! Assertion_Error
```

Comment:
I agree with @JaroslavTulach's suggestion, this should be just `Boolean`
```suggestion
assert : Boolean -> Text -> Nothing ! Assertion_Error
```

If `action : Any -> Boolean`, what's the value of the `Any` argument that would be passed to it?

Reply:
Good point. Fixed in 5f3d6f676183d84aab5bc9975145a37ee0245d38.